benchcab/benchcab.py

```diff
@@ -1,6 +1,3 @@
-# Copyright 2022 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details.
-# SPDX-License-Identifier: Apache-2.0
-
```

Comment:
We shouldn't remove the copyright notice from this file. Can we reinstate please?

Reply:
Sure thing, I think it was when I was trying to get ruff to work, for some reason it didn't like the double "#" comments. Typically a triple quote is used at the top of Python files but I will see how it goes.